tests/integration/targets/lookup_hashi_vault/lookup_hashi_vault/tasks/tests.yml

```diff
@@ -1,76 +1,75 @@
-- name: 'test {{ auth_type }} auth without SSL (lookup parameters)'
+- name: 'test {{ auth_type }} auth{{ ", with proxy" if conn_param_proxy }}, without SSL (lookup parameters)'
   include_tasks: '{{ auth_type }}_test.yml'
   vars:
-    conn_params: 'url=http://localhost:8200 '
+    conn_params: '{{ conn_param_proxy }} url=http://localhost:8200 '
 
-- name: 'test {{ auth_type }} auth without SSL (environment variable)'
+- name: 'test {{ auth_type }} auth{{ ", with proxy" if conn_param_proxy }}, without SSL (environment variable)'
   include_tasks: '{{ auth_type }}_test.yml'
   args:
     apply:
       vars:
-        conn_params: ''
+        conn_params: '{{ conn_param_proxy }} '
       environment:
         VAULT_ADDR: 'http://localhost:8200'
 
 - when: cryptography_version.stdout is version('1.6', '>=')
   block:
-    - name: 'test {{ auth_type }} auth with certs (validation enabled, lookup parameters)'
+    - name: 'test {{ auth_type }} auth{{ ", with proxy" if conn_param_proxy }}, with certs (validation enabled, lookup parameters)'
       include_tasks: '{{ auth_type }}_test.yml'
       vars:
-        conn_params: 'url=https://localhost:8201 ca_cert={{ local_temp_dir }}/cert.pem validate_certs=True '
+        conn_params: '{{ conn_param_proxy }} url=https://localhost:8201 ca_cert={{ local_temp_dir }}/cert.pem validate_certs=True '
```

Comment:
Following on from the comment in `main.yml`, as I look over all these connection tests, at first glance it seems like we'd want to do proxy and no proxy tests for all of these variations, and that may be true in the future, but I need to re-examine all the invocation in here that are using environment variables. As far as I know, those shouldn't be working, so I suspect the env vars are being ignored and we're not testing what we think we are.

In any case, I think for the purposes of testing proxy for now, we should just add 2 invocations here: one using an HTTPS target and one with an HTTP target. So that would mean duplicating this highlighted test, and the first one in the file, to make a proxy variant of each.

At that point, you'd be modifying `conn_params` directly on each of those calls, so we can do away with the `conn_param_proxy` var.

Reply:
done

> In any case, I think for the purposes of testing proxy for now, we should just add 2 invocations here: one using an HTTPS target and one with an HTTP target. So that would mean duplicating this highlighted test, and the first one in the file, to make a proxy variant of each.

I agree